x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/assets/assets.tsx

```diff
@@ -13,9 +13,10 @@ import { groupBy } from 'lodash';
 
 import type { ResolvedSimpleSavedObject } from 'src/core/public';
 
+import { useKibana } from '../../../../../../../../../../../src/plugins/kibana_react/public';
```

Comment:
just swung by this PR to backport it as a patch to 8.0 and noticed this, we have `useStartServices` hook which encapsulates the plugin logic added by this PR so that can be use instead of traversing all the way to the `kibana_react` plugin:
https://github.com/elastic/kibana/blob/f4713f5a26e722f2169483c286ea69a5782f6484/x-pack/plugins/fleet/public/hooks/use_core.ts#L11

I believe you'd just have to add `spaces` to the `FleetStartServices` interface. I think this would be a good candidate for a small PR 😉

Reply:
Amazing thanks! I'll get that done some time soon